src/features/rewards/tableDonation/index.tsx

```diff
@@ -9,15 +9,14 @@ import {
   StyledRemove,
   StyledRemoveIcon,
   StyledToggle,
-  StyledRecurringIcon,
```

Comment:
Can you also remove the unused styled component in `style.ts`?

Reply:
Done.